jekyll/_cci2/circleci-images.md

```diff
@@ -50,6 +45,30 @@ to pin an image to a version or operating system (OS).
 - Use a Docker image ID
 to pin an image to a fixed version.
 
+**NOTE:** For docker images that have `-node` (Node.js) included in the container, the latest LTS
```

Comment:
Docker should be capitalized and `-node` is part of a tag, which forms the image name. The container aspect doesn't exists until it's run.

Maybe something like this?

```
**NOTE:** For Node.js variant Docker images (tags that end in `-node`) the LTS release of Node.js is pre-installed.
```

Reply:
Makes sense! Done.